morphir/runtime/src/org/finos/morphir/runtime/Utils.scala

```diff
@@ -128,10 +129,11 @@ object Extractors {
       case _               => false
     }
   }
+  // Extractor object that unwraps a single layer of aliasing, and gives and type names bound in the process
```

Comment:
"and gives and type names bound in the process"
don't understand

Reply:
"Gives the type names that were bound in the process", but we're not going to get easily comprehensible documentation at this stage in the process

Reply:
Cleaned it up a bit tho, at least

Reply:
much better now